server/routes/referrals/desiredOutcomesForm.test.ts

```diff
@@ -1,89 +1,36 @@
-import { Request } from 'express'
+import TestUtils from '../../../testutils/testUtils'
 import DesiredOutcomesForm from './desiredOutcomesForm'
 
 describe(DesiredOutcomesForm, () => {
-  describe('isValid', () => {
-    it('returns true when the desired-outcomes-ids property is present and not empty in the body', async () => {
-      const form = await DesiredOutcomesForm.createForm({
-        body: {
+  describe('data', () => {
+    describe('when desired outcomes ids are passed', () => {
+      it('returns params for update when the service-category-ids property is present and not empty in the body', async () => {
```

Comment:
I think that we can lose the first part of these `it` statements, which have been moved into the `describe`. So e.g. this one would just be `it('returns params for update')` etc.